app/controllers/spree/admin/orders_controller_decorator.rb

```diff
@@ -83,7 +83,7 @@ def invoice
     template = if Spree::Config.invoice_style2? then "spree/admin/orders/invoice2" else "spree/admin/orders/invoice" end
     pdf = render_to_string pdf: "invoice-#{@order.number}.pdf", template: template, formats: [:html], encoding: "UTF-8"
     Spree::OrderMailer.invoice_email(@order.id, pdf).deliver
-    flash[:success] = t(:invoice_email_sent)
+    flash[:success] = t('admin.orders.invoice_email_sent')
```

Comment:
The only problem is that we're still workin on the step 6 and I can't tell when it'll land in the master branch. If this is not a big problem, we can wait for the step 6 to be done, if not, go ahead with this one.